boa/cli/mambabuild.py

```diff
@@ -34,9 +34,11 @@ def parse_problems(problems):
 
     for line in problems.splitlines():
         line = line.strip()
-        words = line.split()
         if not line.startswith("- "):
             continue
+        if line.startswith("- - "):
```

Comment:
It is actually `"-   - "`, i.e., this missed the additional spaces between the dashes.
In gh-306 I added test cases for this and rewrote the whole parsing logic to use a regular expression to make it a little more robust again these kinds of things.

Reply:
> It is actually "- - ", i.e., this missed the additional spaces between the dashes.

I'm not sure I understand. Aren't they the same thing?

Reply:
Argh, Markdown formatter squeezed those spaces together in that inline code block.
```
"-   - "
```
was what I meant.
(In gh-306 the single commits ought to show the single failing steps, but the CI didn't run for those since I hadn't contributed to this repository before; "First-time contributors need a maintainer to approve running workflows.")